### PR TITLE
add event signin options page

### PR DESCRIPTION
**Pull request by audreygao**

### CODE REVIEW

File: frontend/src/pages/EventDetailsPage/components/AttendanceTable/index.tsx [hunk 1]
```diff
@@ -1,14 +1,15 @@
 import React, { useEffect, useState } from 'react';
 import { parseISO, format } from 'date-fns';
 
+import AttendanceDeleteButton from '../buttons/AttendanceDeleteButton';
```
Comment (thaigillespie) [inline frontend/src/pages/EventDetailsPage/components/AttendanceTable/index.tsx:4]: You didn't change this did you? I'm guessing this was Prettier just doing its job? I see all the files related to delete attendance have their formatting changed, but not the actual content itself

**Review by thaigillespie — APPROVED** [review]
Lgtm, I do have one question for you that I want to be resolved before you merge this in

File: frontend/src/pages/EventDetailsPage/components/AttendanceTable/index.tsx (hunk 1 above)
Comment (audreygao) [reply frontend/src/pages/EventDetailsPage/components/AttendanceTable/index.tsx:4]: I didn't change it. I think vs code somehow reformatted those and not the content.

**Comment by thaigillespie** [comment]
Alright sounds good

**Pull request merged**